app/data/models/internal_orders.py

```diff
@@ -9,8 +9,7 @@ class InternalOrder(Base):
 
     id = sa.Column(sa.Integer, primary_key=True)
     lead_time = sa.Column(sa.DateTime, nullable=False)
-    products = relationship('Product', back_populates='internal_order')
-    #  TODO: Make many-> many relations with products and internal_orders
+    products_has_internal_orders = relationship('ProductInternalOrders', back_populates='internal_order')
```

Comment:
internal_orders instead of internal_order on back_populates

Reply:
fixed: https://github.com/AntonMaxen/Databas01/pull/41/commits/4c1bb97a6e5210819260679e496e8894d795ae95